### PR TITLE
Statistics page

### PR DESCRIPTION
**Pull request by lamnhh**
Add more information on /stats. More specifically:

1. A ranking on users based on number of posts they made.
2. A chart that shows how many posts was made during each day during the last 7 days.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -85,28 +97,71 @@ public void storeMessage(Message message) {
...
     return results.countEntities(FetchOptions.Builder.withLimit(1000));
   }
+
+  /**
+   * Get top 10 users with most posts
+   * 
+   * @return a list of at most 10 users with most posts, each element is in the
+   *         form [post_count, user_name
+   */
+  public JsonArray getRanking() {
```
Comment (quanglm1998) [inline src/main/java/com/google/codeu/data/Datastore.java:131]: This function returns all users, not just at most 10 users!

File: src/main/webapp/js/stats-loader.js [hunk 2]
```diff
@@ -1,27 +1,105 @@
...
-      } in total`;
-      statsContainer.innerHTML += `<div class="post">
-        <div class="post-header">
-          <h2 class="post-uploader">Message count</h2>
-        </div>
-        <div class="post-content">
-          ${msg}
+async function fetchStats() {
+  const statsContainer = document.getElementsByClassName("public-feed")[0];
+  const stats = await fetch("http://localhost:8080/stats").then((response) =>
```
Comment (quanglm1998) [inline src/main/webapp/js/stats-loader.js]: I think it should be just "/stats".

**Comment by lamnhh** [comment]
@quanglm1998 @anhducle98 I just fixed it. Please review the PR again.

**Review by quanglm1998 — APPROVED** [review]
Everything seems good!

**Pull request merged**